examples/ConvOpt/pointwise_conv2d_nhwc_filter_hwcf.mlir

```diff
@@ -1,9 +1,6 @@
 // Generated from Mobilenet.mlir file
 func.func @conv_2d_1x1(%input: tensor<1x4x5x2xf32>, %filter: tensor<1x1x2x7xf32>) -> tensor<1x4x5x7xf32> {
     %0 = linalg.init_tensor [1, 4, 5, 7] : tensor<1x4x5x7xf32>
-    %1 = linalg.conv_2d_nhwc_hwcf {
```

Comment:
What's the purpose of this modification?

Reply:
This modification will be undone.